test/constants/types.js

```diff
@@ -0,0 +1,42 @@
+const VAULTS_CATEGORY = [
```

Comment:
please just add a link to https://github.com/harvest-finance/harvest-api/blob/master/data/constants.js , it is not necessary to copy all the data here, it would require manual update.